README.md

```diff
@@ -341,7 +341,7 @@ function Profile () {
         // send a request to the API to update the data
         await requestUpdateUsername(newName)
         // update the local data immediately and revalidate (refetch)
-        mutate('/api/user', { ...data, name: newName })
+        mutate({ ...data, name: newName })
```

Comment:
should this line have changed? as this isn't a bound `mutate`

Reply:
I think we should add a comment which mentions that this is the global `mutate`.

Reply:
Yes, my mistake. Will fix it.